docs/CI_PIPELINE.md

```diff
@@ -0,0 +1,80 @@
+## CI Pipeline
+
+### Decisions on CI product and style
+
+We are currently using github actions for our CI pipeline. We prefer this over circleci as it is
```

Comment:
Phrasing here makes it more an ADR, i.e. decision to use GitHub. Might be worth moving there.